README.md

```diff
@@ -48,14 +54,18 @@ module.exports = {
             use: [
                 "style-loader", // creates style nodes from JS strings
                 "css-loader", // translates CSS into CommonJS
-                "sass-loader" // compiles Sass to CSS
+                {
+                  loader: "sass-loader", // compiles Sass to CSS
+                  // require("node-sass") instead to use Node Sass
+                  options: {sass: require("sass")}
```

Comment:
Should not be option. `sass-loader` should resolve `node-sass`, then try to resolve `dart-sass`, if not found nothing - throw error

Reply:
Are you sure? Not allowing users to explicitly control this will mean that if they (or one of their dependencies if they're using flat dependencies) requires Node Sass for some unrelated reason, they're forced to use Node Sass. This may even result in them unexpectedly switching to Node Sass down the line.

It seems like [several](https://github.com/webpack-contrib/sass-loader/issues/435#issuecomment-391853396) [users](https://github.com/webpack-contrib/sass-loader/issues/435#issuecomment-391855181) would prefer making this configuration explicit.

Reply:
@nex3 Hm, yes you are right it can lead to problems, let's rename `sass` to `implementation` and try to load `node-sass` by default. It is allow to upgrade `loader` without break build and avoid unnecessary `major`.

Reply:
Done. It's worth noting though that the name `implementation`, in addition to being more of a pain to type, means that users can't just write

```js
const sass = require("sass");

// ...
    options: { sass }
// ...
```